lib/gym/xcodebuild_fixes/watchkit2_fix.rb

```diff
@@ -0,0 +1,35 @@
+module Gym
+  class XcodebuildFixes
+    class << self
+      # Determine whether this app has WatchKit2 support and manually package up the WatchKit2 framework
+      def watchkit2_fix
+        return unless watchkit2?
```

Comment:
I wonder if this could also be something like `return unless watchkit2? and xcode6?`?